### DISCUSSION TITLE
Make frontend a progressive webapp

### DISCUSSION THREAD

**Discussion by neroes** [Kareadita/Kavita#3152] [Ideas]
### Idea Description

1. It is nice with an app "bookmark" to something that we use so much on tablets and other reading devices.
2. The local copy of the frontend doesn't have to load each time making it a much smoother experience when bandwidth is a constrain.
3. Progressive webapps don't have to deal with the address bar, and i can be a hazzle when you scroll up a bit it re apears.
4. Currently no app is anywhere near as feature complete with the API as Kavita's own frontend.
5. A further expansion could allow for offline usage with "cached" books and comics.

### Idea Category

Web UI

### Duration of Using Kavita

very new

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Kavita is a progressive webapp. the only thing it's missing is the offline functionality.
